### DISCUSSION TITLE
Video upload fails

### DISCUSSION THREAD

**Discussion by yurividal** [immich-app/immich#14153] [Q&A] [answered]
### The bug

I have a video that keeps failing to upload.

Logs:

![Screenshot_20241114_191355](https://github.com/user-attachments/assets/b04de558-3c8a-4ddf-8191-ac62f3ff7411)

### The OS that Immich Server is running on

Ubuntu 22.04 LTS

### Version of Immich Server

v1.120.2

### Version of Immich Mobile App

v1.120.2

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Similar to examples
```

### Your .env content

```Shell
Similar to examples
```

### Reproduction steps

Take photos and videos.
Video fails to upload

### Additional information

I have recently moved from nginx-proxy-manager to npmPlus as my proxy. It's still nginx under the hood, so I don't think this could be the issue, but it's the only thing that has changed in the past days, apart from updating immich to latest versions.

**Comment by bo0tzz** [comment]
Make sure your reverse proxy isn't limiting the request body size. https://immich.app/docs/administration/reverse-proxy

**Reply by yurividal** [reply]
This is my current npm advanced condig for immich:

̀ ̀ ̀ ̀
    set_real_ip_from 172.29.0.0/16;
    set_real_ip_from 103.21.244.0/22;
    set_real_ip_from 103.22.200.0/22;
    set_real_ip_from 103.31.4.0/22;
    set_real_ip_from 104.16.0.0/12;
    set_real_ip_from 108.162.192.0/18;
    set_real_ip_from 131.0.72.0/22;
    set_real_ip_from 141.101.64.0/18;
    set_real_ip_from 162.158.0.0/15;
    set_real_ip_from 172.64.0.0/13;
    set_real_ip_from 173.245.48.0/20;
    set_real_ip_from 188.114.96.0/20;
    set_real_ip_from 190.93.240.0/20;
    set_real_ip_from 197.234.240.0/22;
    set_real_ip_from 198.41.128.0/17;
    real_ip_header X-Forwarded-For;
    real_ip_recursive on;
    proxy_set_header X-Real-IP $remote_addr;

    # allow large file uploads
    client_max_body_size 50000M;

    # Set headers
    proxy_set_header Host              $http_host;
    proxy_set_header X-Real-IP         $remote_addr;
    proxy_set_header X-Forwarded-For   $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;

    # enable websockets: http://nginx.org/en/docs/http/websocket.html
    proxy_http_version 1.1;
    proxy_set_header   Upgrade    $http_upgrade;
    proxy_set_header   Connection "upgrade";
    proxy_redirect     off;

    # set timeout
    proxy_read_timeout 600s;
    proxy_send_timeout 600s;
    send_timeout       600s;

    location / {
        proxy_pass http://immich-server:2283;
    }


` ̀ ̀ ̀

**Comment by alextran1502** [comment]
I've seen some reports about this. Probably due to dependencies update, this is manual upload, correct?

**Reply by yurividal** [reply]
Yes, manual upload after normal upload failed

**Comment by yurividal** [comment, marked as answer]
Managed to solve this by disabling modsecurity on npm-plus for the immich host
